README.md

```diff
@@ -459,6 +460,18 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Pidgin](https://www.pidgin.im)
 - [Convey](https://bitbucket.org/rw_grim/convey)
 
+### Geoffrey Huntley
+#### What Geoff Streams
+- Haskell, Reactive Extensions, NixOS, OSS Development, C#, .NET, Xamarin, DevOps
+#### Streaming on:
+- [Twitch](https://www.twitch.tv/geoffreyhuntley)
+- [Twitter](https://twitter.com/geoffreyhuntley)
```

Comment:
Could you move Twitter down to the links section? Twitter doesn't do any kind of streaming, right?

Reply:
![image](https://user-images.githubusercontent.com/127353/52936375-099c2700-33b0-11e9-879c-6150d490674b.png)

Twitter supports streams. See above. I simulcast on Twitch, Twitter and YouTube.

Reply:
Wow, I had no idea they did that. Thanks for educating me!